frontend-react/src/components/annotate.js

```diff
@@ -41,17 +42,30 @@ class Annotate extends Component {
         }
 
         return (
-            <div className="backgroundAll">
+            <div className="border">
                 <form className="selectForm" onSubmit={this._handleSubmit}>
                     <input type="file" id="file" className="fileInputButton" onChange={this._handleImageChange} />
                     <label for="file" class="fileLabel">Select Image</label>
-                    {/* <button type="submit" onClick={this._handleSubmit}>Upload Image</button> */}
+                    {/* <button type="submit" onClick={this._handleSubmit}>Send Annotations</button> */}
```

Comment:
Not sure what the point of this is. Most of this should probably be removed anyway due to the use of the library right?

Reply:
Was left in the code incase we decide not to use the library. Still haven't confirmed we want to use the library until we can understand the way to get the annotation data. Have removed it for you as it shouldn't be too hard to remember.